### DISCUSSION TITLE
I removed my primary email at github by mistake and now I can't login

### DISCUSSION THREAD

**Discussion by zobkazi** [community/community#78009] [New to GitHub] [answered]
Due to my little mistake I lost my account there are my important documents I made a mistake by removing the primary email now I can't login to my account any way and no access from the account how can I login to my account

**Comment by amaar-mc** [comment, marked as answer]
Hey @zobaidulkazi64, if you remember your account name and your repositories are public then you can try looking at the profile and retrieving your files onto a new profile. I don't believe GitHub allows you to remove ALL emails when deleting an email because they need to have an email to allow you to login, so try seeing if there are any other emails you use for GitHub. If not, try remembering your password and your username, and see if you linked your phone number possibly. If you found this comment help, please mark it as "answered" by the way, hope I helped!

**Reply by zobkazi** [reply]
I have kept all as backup no problem but it is not possible to get back and my big mistake was that I unlike my email from the account that's why there is no way to login with email don't give username

**Comment by zobkazi** [comment]
Perhaps I made another mistake, I unlike the email, so the email will be completely removed from there, now there is no way to go from the email.

My account is still active no way to login both email and username are showing wrong so will it be a lifetime or will it be deleted once again I will open again now
